CHANGELOG.md

```
@@ -23,6 +23,33 @@
 * accept 201 greetings response on NNTP connection (Fixes [#104](https://github.com/Gaisberg/streamnzb/issues/104)) ([5cd8bf4](https://github.com/Gaisberg/streamnzb/commit/5cd8bf49490874fbe87f0baa58f5915f3ce95c43))
 
 
+### Miscellaneous Chores
+
+* release 4.0.1 ([0a4e5f7](https://github.com/Gaisberg/streamnzb/commit/0a4e5f7d82cbc5c912ff96f8793cc8342ef7ef30))
+
+## [4.0.1](https://github.com/Gaisberg/streamnzb/compare/v3.7.0...v4.0.1) (2026-04-09)
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)